docs/admin/external-auth.md

```diff
@@ -212,8 +212,32 @@ you can require users authenticate via git prior to creating a workspace:
 
 ![Git authentication in template](../images/admin/git-auth-template.png)
 
-The following example will require users authenticate via GitHub and auto-clone
-a repo into the `~/coder` directory.
+### `GIT_ASKPASS` will auto-refresh tokens
```

Comment:
I think users will be unfamiliar with what `GIT_ASKPASS` is.

We should approach this from a `git` perspective... as the tool most users are familiar with, and explain that we use `GIT_ASKPASS` is a variable Git hooks into for authentication.

It's worth mentioning that `coder external-auth token <id>` will output the same token as the Git hook will.

Reply:
Will update.